LEGO1/lego/legoomni/src/video/legovideomanager.cpp

```diff
@@ -125,14 +125,18 @@ MxResult LegoVideoManager::Create(MxVideoParam& p_videoParam, MxU32 p_frequencyM
 
 	m_direct3d->SetDevice(deviceEnumerate, driver, device);
 
-	if (!driver->m_ddCaps.dwCaps2 && driver->m_ddCaps.dwSVBRops[7] != 2) {
-		p_videoParam.Flags().SetF2bit0(TRUE);
+	/*
+	 * BUG: should be:
+	 *  if ((driver->m_ddCaps.dwCaps2 & DDCAPS2_CERTIFIED) == 0 && driver->m_ddCaps.dwSVBRops[7] != 2) {
+	 */
+	if (driver->m_ddCaps.dwCaps2 != DDCAPS2_CERTIFIED && driver->m_ddCaps.dwSVBRops[7] != 2) {
```

Comment:
Unfortunately this does decrease the assembly accuracy:

```
Decreased (1):
0x1007ac40 - LegoVideoManager::Create (98.63% -> 98.17%)
```

![image](https://github.com/user-attachments/assets/0ad9d3ad-2363-4ebd-8e34-a1d6a65ac5bc)

Reply:
Maybe they really did check against it being empty as a way to tell if it's a DirectX 5 feature level driver or something.

Reply:
I'm not going to juggle the ways this could be written or setup the needed tools to make a comparable version so I have reduced this PR to just be the name change for the flag.